hist/hist/src/TPrincipal.cxx

```diff
@@ -898,8 +895,8 @@ void TPrincipal::MakeRealCode(const char *filename, const char *classname,
                               Option_t *)
 {
    Bool_t  isMethod = (classname[0] == '\0' ? kFALSE : kTRUE);
-   const char *prefix   = (isMethod ? Form("%s::", classname) : "");
-   const char *cv_qual  = (isMethod ? "" : "static ");
+   TString prefix   = isMethod ? TString::Format("%s::", classname) : TString("");
```

Comment:
```suggestion
   TString prefix;
   if (isMethod)
   	prefix.Form("%s::", classname);
```

Reply:
Done in https://github.com/root-project/root/pull/13438/commits/f7e55aba4e52ffec8caf1c03a83403b35c7f4369